tests/shell/shell_recorder/listtest.esr

```diff
@@ -16,4 +16,5 @@ RET: 0
 
 < kdb ls $Mountpoint
 
+< kdb rm /sw/kdb/current/plugins
```

Comment:
> Please create an issue or add it to #401.

Since I do not think that issue #401 describes the same problem, I just opened a new issue [here](https://github.com/ElektraInitiative/libelektra/issues/1483).

> Did you also try `kdb mv`?

Yes, but I already stopped after `kdb mv` reported the following problem:

```
First key found user/sw/kdb/current/plugins does not exactly match given key
/sw/kdb/current/plugins, aborting (use -r to move hierarchy)
```
after I entered

```sh
kdb mv /sw/kdb/current/plugins user/backup
```

. 

> I am afraid you have to do import/export (cascading currently not working with all plugins, see #51) or copy all namespaces separately (using `kdb cp`).

Honestly, that sounds like too much effort for a simple test. I just removed the two commits that changed `listtest.esr`. That should get rid of the possibility that the test removes user data.